MAINTAIN.md

```diff
@@ -76,6 +76,15 @@ These "bundled" dependencies can be updated by bumping their versions in `third-
   - [lua-compat](https://github.com/keplerproject/lua-compat-5.3)
   - [tree-sitter](https://github.com/tree-sitter/tree-sitter)
 
+`scripts/bump-dep.sh` is a script that can automate this process for `LuaJIT`, `Luv`, `libuv` & `tree-sitter`. See usage guide:
+  - Switch to a branch, with prefix `bump-`
+  Use `./scripts/bump-deps.sh --branch foo` to create branch `bump-foo` from `master`
```

Comment:
Can this be made automatic (i.e., on `--dep --version`)? It's OK to have an explicit option, but the real benefit is if it happens automatically.

(The docs here can focus on the "minimal" case of bumping a single dependency without manual commits.)

Reply:
Done. Should the explicit branch creation be removed?

Reply:
Thanks! It's ok to leave it in; you never know when it'll be useful.